nixos/modules/programs/projecteur.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.projecteur;
+in
+{
+  options.programs.projecteur = {
+    enable = lib.mkEnableOption (lib.mdDoc "projecteur");
+    package = lib.mkPackageOptionMD pkgs "projecteur" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment = {
+      systemPackages = [ cfg.package ];
+      services.udev.packages = [ cfg.package ];
```

Comment:
this line should be on the same level as environment directly below config

Reply:
I don't understand. Are you asking to change the indentation?

Reply:
no the option environment.services does not exist, the code should be
```
    environment.systemPackages = [ cfg.package ];
    services.udev.packages = [ cfg.package ];
```

Reply:
Oh ! nice catch, just fixed it. Thanks.